build/ios-uitest-run.sh

```diff
@@ -38,6 +38,7 @@ else
 		namespace = 'SamplesApp.UITests.Windows_UI_Xaml_Media.Animation_Tests' or \
 		namespace = 'SamplesApp.UITests.Windows_UI_Xaml_Controls.ControlTests' or \
 		namespace = 'SamplesApp.UITests.Windows_UI_Xaml_Controls.TextBlockTests' or \
+		namespace = 'SamplesApp.UITests.Windows_UI_Xaml_Controls.TextBoxTests' or \
```

Comment:
I see that `TextBoxTests` has been added here, Since `PivotTests` was also updated in this PR, do they also belong here?

Reply:
Added. PivotTests

Reply:
Correctly adjusted namespace after rebase from master